### DISCUSSION TITLE
G14 2020 Charging

### DISCUSSION THREAD

**Discussion by Akalikaton** [seerge/g-helper#710] [Q&A]
First of all thanks for your Work, so that this one is much liter than offical. 

I use G14 2020, when I use WIN11 22H2, my powerbank can charge it use 9V 2A(it doesn't work on WIN10). Even though it doesn't show it's charging on the computer, the battery will slowly rise under low load. However, after using this great software, the power bank is unable to charge. Just as I don't know why it was able to charge, I don't know why it can't charge now. 

In addition, I would like to ask under what condition can the Nintendo Switch charger be used, which can output 15V 2.6A, and some people claim it can also output 45W PD.

**Comment by Akalikaton** [comment]
PS: a new try, it can be charged with 9V 2A and the charging light is on

**Comment by seerge** [comment]
Hello, I honestly don't know why your power bank can't charge :) it's controlled by your hardware not by the apps. 

As a general advice, just make sure that your laptop uses less power than it can potentially get from charger in order to charge :) I.e. if you use it, set Silent / Eco modes to minimise power consumption.

**Reply by Akalikaton** [reply]
Hello, I'm glad to hear your response. 

I heard that Mac can be charged with 20W PD, which surprised me that the ROG G14 can also use it. Typically, gaming laptops can only be charged with PD higher than 65W, and in fact, they cannot be used in the Windows 10 system, but they can be used in the Windows 11 system. 

If I use it for outside, I will enable silent mode, limit the CPU to a maximum of 50%, and set the TDP to 15W. Under these conditions, if I'm just working with documents and browsing the web, the power consumption displayed by Ghelper would be around 7W. By also using my 110Wh power bank, I have never completely used all the battery power.
